spec.html

```diff
@@ -33991,24 +33991,20 @@ <h1>
                 1. Let _refReplacement_ be the substring of _str_ from min(_tailPos_, _stringLength_).
                 1. NOTE: _tailPos_ can exceed _stringLength_ only if this abstract operation was invoked by a call to the intrinsic @@replace method of %RegExp.prototype% on an object whose *"exec"* property is not the intrinsic %RegExp.prototype.exec%.
               1. Else if _templateRemainder_ starts with *"$"* followed by 1 or more decimal digits, then
-                1. Let _found_ be *false*.
-                1. For each integer _d_ of &laquo; 2, 1 &raquo;, do
-                  1. If _found_ is *false* and _templateRemainder_ starts with *"$"* followed by _d_ or more decimal digits, then
-                    1. Set _found_ to *true*.
-                    1. Let _ref_ be the substring of _templateRemainder_ from 0 to 1 + _d_.
-                    1. Let _digits_ be the substring of _templateRemainder_ from 1 to 1 + _d_.
-                    1. Let _index_ be ℝ(StringToNumber(_digits_)).
-                    1. Assert: 0 &le; _index_ &le; 99.
-                    1. If _index_ = 0, then
-                      1. Let _refReplacement_ be _ref_.
-                    1. Else if _index_ &le; the number of elements in _captures_, then
-                      1. Let _capture_ be _captures_[_index_ - 1].
-                      1. If _capture_ is *undefined*, then
-                        1. Let _refReplacement_ be the empty String.
-                      1. Else,
-                        1. Let _refReplacement_ be _capture_.
-                    1. Else,
-                      1. Let _refReplacement_ be _ref_.
+                1. If _templateRemainder_ starts with *"$"* followed by 2 or more decimal digits, let _digitCount_ be 2. Otherwise, let _digitCount_ be 1.
+                1. Let _ref_ be the substring of _templateRemainder_ from 0 to 1 + _digitCount_.
```

Comment:
Can we define an alias for `1 + _digitCount_` so it reads more clearly?

Reply:
Yes, but the obvious name for that is _refLength_, which already has a Let step after this If...Else if...Else series. Should I adopt the suggestion of https://github.com/tc39/ecma262/pull/2739#discussion_r850845182 , using length rather than substring in every case?

Reply:
That WFM. @tc39/ecma262-editors?

Reply:
No alias sgtm.